groups/sig-testing/groups.yaml

```diff
@@ -182,7 +182,19 @@ groups:
     settings:
       ReconcileMembers: "true"
       WhoCanPostMessage: "ANYONE_CAN_POST"
-      WhoCanViewMembership: "ALL_MEMBERS_CAN_VIEW" # required
+      WhoCanViewMembership: "ALL_MEMBERS_CAN_VIEW"
+    managers:
+    - ameukam@gmail.com
+    - spiffxp@gmail.com
+    - spiffxp@google.com
+
+  - email-id: k8s-infra-cherrypick-robot@kubernetes.io
```

Comment:
@spiffxp Will be great to see the password for this account in a 1password vault.